src/notifications/index.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+
+import Icon from "@reearth/components/atoms/Icon";
+import Text from "@reearth/components/atoms/Text";
+import Flex from "@reearth/components/atoms/Flex";
+
+import { styled, metrics, useTheme } from "@reearth/theme";
+import useHooks, { NotificationType as Type, NotificationStyleType as StyleType } from "./hooks";
+
+export type NotificationType = Type;
+export type NotificationStyleType = StyleType;
+
+const NotificationBanner: React.FC = ({ children }) => {
+  const theme = useTheme();
+  const { visible, changeVisibility, notification, setNotification } = useHooks();
+
+  return (
+    <div style={{ height: "100%" }}>
+      <StyledNotificationBanner visible={visible} type={notification?.type} direction="column">
+        <HeadingArea justify="space-between">
+          <Text
+            size="m"
+            color={theme.notification.text}
+            weight="bold"
+            otherProperties={{ padding: "0 0 8px 0" }}>
+            {notification?.heading}
+          </Text>
+          <CloseBtn
+            icon="cancel"
+            size={20}
+            onClick={() => {
+              changeVisibility(false);
+              setNotification(undefined);
+            }}
+          />
+        </HeadingArea>
+        <Text size="s" color={theme.notification.text}>
+          {notification?.text}
+        </Text>
+      </StyledNotificationBanner>
+      {children}
+    </div>
```

Comment:
This component doesn't need to have children, otherwise the component may become too complex:

```suggestion
  <StyledNotificationBanner visible={visible} type={notification?.type} direction="column">
    <HeadingArea justify="space-between">
      <Text
        size="m"
        color={theme.notification.text}
        weight="bold"
        otherProperties={{ padding: "0 0 8px 0" }}>
        {notification?.heading}
      </Text>
      <CloseBtn
        icon="cancel"
        size={20}
        onClick={() => {
          changeVisibility(false);
          setNotification(undefined);
        }}
      />
    </HeadingArea>
    <Text size="s" color={theme.notification.text}>
      {notification?.text}
    </Text>
  </StyledNotificationBanner>
```